tests/topotests/example_munet/test_munet.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 eval: (blacken-mode 1) -*-
+#
+# April 23 2023, Christian Hopps <chopps@labn.net>
```

Comment:
Let's add SPDX-License tag for example \m/

Reply:
oops, fixed.